### ISSUE TITLE
`now --alias` prints `--help` output after completing

### ISSUE THREAD

**Issue by mathiasbynens** [vercel/vercel#272]
```
$ now -v
𝚫 now 4.0.1

$ now --alias
> Deploying ~/projects/jsperf.com
> Using Node.js 6.9.4 (requested: `6.x.x`)
> Ready! https://[hidden].now.sh (copied to clipboard) [2s]
> Initializing…

  𝚫 now [options] <command | path>

  Commands:

    deploy       [path]       Performs a deployment (default)
    ls | list    [app]        List deployments
    rm | remove  [id]         Remove a deployment
    ln | alias   [id] [url]   Configures aliases for deployments
    domains      [name]       Manages your domain names
    certs        [cmd]        Manages your SSL certificates
    secrets      [name]       Manages your secret environment variables
    dns          [name]       Manages your DNS records
    help         [cmd]        Displays complete help for [cmd]

  Options:

    -h, --help                Output usage information
    -v, --version             Output the version number
    -n, --name                Set the name of the deployment
    -c FILE, --config=FILE    Config file
    -d, --debug               Debug mode [off]
    -f, --force               Force a new deployment even if nothing has changed
    -t TOKEN, --token=TOKEN   Login token
    -L, --login               Configure login
    -l, --links               Copy symlinks without resolving their target
    -p, --public              Deployment is public (`/_src` is exposed) [on for oss, off for premium]
    -e, --env                 Include an env var (e.g.: `-e KEY=value`). Can appear many times.
    -C, --no-clipboard        Do not attempt to copy URL to clipboard
    -N, --forward-npm         Forward login information to install private npm modules
    -a, --alias               Re-assign existing aliases to the deployment

  Enforcable Types (when both package.json and Dockerfile exist):

    --npm                     Node.js application
    --docker                  Docker container
    --static                  Static file hosting

  Examples:

  – Deploys the current directory

    $ now

  – Deploys a custom path `/usr/src/project`

    $ now /usr/src/project

  – Deploys a GitHub repository

    $ now user/repo#ref

  – Deploys a GitHub, GitLab or Bitbucket repo using its URL

    $ now https://gitlab.com/user/repo

  – Deploys with ENV vars

    $ now -e NODE_ENV=production -e MYSQL_PASSWORD=@mysql-password

  – Displays comprehensive help for the subcommand `list`

    $ now help list

> Deployment complete!
```

Note how it also doesn’t display which domain it has aliased to. 

Might be related to add7bcfbea587af68e88e6096f93f572beebdfcd.

**Comment by leo** [comment]
Can I see your alias config please?

Should be an `aliases` property containing an array. If that's not your intention, here are the details: Leaving `--alias` empty uses the `aliases` property in your config and applies all of the aliases in there. If you only want to use a certain domain, use `--alias zeit.co`.

Maybe you ignored it, but we did a major release because of that... 😊 

[More](https://zeit.co/blog/now-json#`aliases`-(array))

**Comment by matheuss** [comment]
Related: https://github.com/zeit/now-cli/commit/7ccb4de336ffd7bbe2aebba0beabc01977cf6546

**Comment by mathiasbynens** [comment]
> Should be an aliases property containing an array.

Ah, my `package.json` still had the old `alias` (not `aliases`) property, like this:

```
  "now": {
    "alias": "jsperf.com"
  }
```

I’ve now updated it [per your instructions](https://zeit.co/blog/now-json#%60aliases%60-(array)):

```
  "now": {
    "aliases": ["jsperf.com"]
  }
```

However, `now --alias` seems to perform a new deployment, and then alias. Previously (before v4) I could run `now alias` after a successful deployment to do just the aliasing — what’s the new command for that?

**Comment by leo** [comment]
Still the same 😊

- `now --alias` is and was always for triggering a new deployment and aliasing it automatically
- `now alias` let's you alias an existing deployment (run `now alias -h` for help)

**Comment by mathiasbynens** [comment]
Well, that’s what led to me filing this issue. Previously, running just `now alias` would alias the most recent deployment using the `alias` field in `package.json`. Since v4 that doesn’t work anymore.

I just tested it with the `aliases` field instead (per the above) but it still errors:

```
$ now alias
> Error! Please specify a valid subcommand: ls | set | rm

  𝚫 now alias <ls | set | rm> <deployment> <alias>

  Options:

    -h, --help              Output usage information
    -c FILE, --config=FILE  Config file
    -d, --debug             Debug mode [off]
    -t TOKEN, --token=TOKEN Login token

  Examples:

  – Lists all your aliases:

      $ now alias ls

  – Adds a new alias to my-api.now.sh:

      $ now alias set api-ownv3nc9f8.now.sh my-api.now.sh

      The `.now.sh` suffix can be ommited:

      $ now alias set api-ownv3nc9f8 my-api

      The deployment id can be used as the source:

      $ now alias set deploymentId my-alias

      Custom domains work as alias targets:

      $ now alias set api-ownv3nc9f8.now.sh my-api.com

      – The subcommand `set` is the default and can be skipped.
      – `http(s)://` in the URLs is unneeded / ignored.

  – Removing an alias:

      $ now alias rm aliasId

      To get the list of alias ids, use `now alias ls`.

  Alias: ln
```

TL;DR `now alias` (without further parameters) regressed in v4. Should I file a new issue for that?

**Comment by leo** [comment]
@mathiasbynens Yep, please do! We need to bring it back. Got removed accidentally, I'd say.
